packages/mc-stats/src/class-mc-stats.php

```diff
@@ -0,0 +1,182 @@
+<?php
+/**
+ * Jetpack MC Stats package.
+ *
+ * @package  automattic/jetpack-mc-stats
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Class MC Stats, used to record stats using https://pixel.wp.com/g.gif
+ */
+class Mc_Stats {
+
+	/**
+	 * Holds the stats to be processed
+	 *
+	 * @var array
+	 */
+	private $stats = array();
+
+	/**
+	 * Indicates whether to use the transparent pixel (b.gif) instead of the regular smiley (g.gif)
+	 *
+	 * @var boolean
+	 */
+	public $use_transparent_pixel = false;
+
+	/**
+	 * Class Constructor
+	 *
+	 * @param boolean $use_transparent_pixel Use the transparent pixel instead of the smiley.
+	 */
+	public function __construct( $use_transparent_pixel = false ) {
```

Comment:
What would you think about defaulting to the invisible pixel? As this doesn't add anything for the site owner—only helps us—I don't see the advantage of defaulting to something that adds visually to the site. Especially since further down we display:none it.

Reply:
it makes sense to me. I was just keeping the current behavior